components/dropdown/dropdown-content-styles.js

```diff
@@ -237,35 +258,47 @@ export const dropdownContentStyles = css`
 		animation: d2l-dropdown-mobile-tray-left-close 300ms ease-out;
 	}
 
+	:host([data-mobile][mobile-tray="bottom"][opened]) > .d2l-dropdown-content-position > .d2l-dropdown-content-width[data-closing] {
+		animation: d2l-dropdown-mobile-tray-bottom-close 300ms ease-out;
+	}
+
+
 	:host([data-mobile][mobile-tray="left"][opened]) > .d2l-dropdown-content-position > .d2l-dropdown-content-width > .d2l-dropdown-content-container,
 	:host([data-mobile][mobile-tray="right"][opened]) > .d2l-dropdown-content-position > .d2l-dropdown-content-width > .d2l-dropdown-content-container {
 		height: 100vh;
 	}
 
 	:host([data-mobile][mobile-tray="left"]) > .d2l-dropdown-content-pointer,
-	:host([data-mobile][mobile-tray="right"]) > .d2l-dropdown-content-pointer {
+	:host([data-mobile][mobile-tray="right"]) > .d2l-dropdown-content-pointer,
+	:host([data-mobile][mobile-tray="bottom"]) > .d2l-dropdown-content-pointer {
```

Comment:
If the value of `mobile-tray` doesn't matter, you could rewrite a lot of these to be:

```css
:host([data-mobile][mobile-tray]) > .d2l-dropdown-content-pointer {
}
```

Hmm, unless that causes the specificity of the CSS rule to be lower and therefore not take precedence?